backend/hitas/models/apartment.py

```diff
@@ -44,29 +44,32 @@ class Apartment(ExternalHitasModel):
     stair = models.CharField(max_length=16)
 
     # 'Luovutushinta'
-    debt_free_purchase_price = models.PositiveIntegerField(default=0)
+    debt_free_purchase_price = models.PositiveIntegerField(null=True)
```

Comment:
I guess `default=0` could be enough (especially if in the old system it defaulted to that), but what I was worried about was:  is it always the correct information?

If an apartment can be created without entering e.g. `primary_loan_amount` and it defaults to zero, how do we tell it apart from something that actually has `primary_loan_amount=0`? or is there even a need to separate the two cases ever?

Reply:
It's a good question do we want to separate them. Old system did not and and it might make calculations a bit easier. We don't have to worry about `NULL`s.

When we have `NULL`:
 - When dealing with basic arithmetic functions like `+` and  `*` will result to `NULL`. (`SELECT 1+NULL` -> `NULL`)
 - When dealing with aggregate functions like `SUM` or `AVG` then `NULL`s are ignored. This does not matter that much to `SUM` but it has an effect to `AVG` (`AVG(<1, 2, 3, NULL, 5>)` is `11/4` instead of `11/5`.
 
 We discussed this offline: let's keep this and update the tests to include new NULL values